joint_trajectory_generator/CMakeLists.txt

```diff
@@ -11,12 +11,13 @@ find_package(catkin REQUIRED COMPONENTS
 
 find_package(Boost REQUIRED COMPONENTS thread)
 find_package(orocos_kdl REQUIRED)
+#pkg_check_modules(OROCOS REQUIRED orocos_kdl)
 
 catkin_package(
   INCLUDE_DIRS include
   LIBRARIES joint_trajectory_generation
   CATKIN_DEPENDS roscpp urdf angles actionlib joint_trajectory_action pr2_controllers_msgs
-  DEPENDS Boost orocos_kdl)
+  DEPENDS Boost liborocos-kdl-dev)
```

Comment:
That looks wrong to me. assuming `find_package(orocos_kdl)` above still declares `${orocos_kdl_INCLUDE_DIRS}` and `*_LIBRARIES` (as they are used below) you should not touch this line either. It simply records the values of these variables.

Reply:
I don't know why, but orocos_kdl no longer resolves in noetic. liborocos-kdl-dev seems to be the new dependency for the same needs as orocos_kdl provided